src/main/java/frc/robot/subsystems/Bling.java

```diff
@@ -0,0 +1,35 @@
+/*----------------------------------------------------------------------------*/
+/* Copyright (c) 2019 FIRST. All Rights Reserved.                             */
+/* Open Source Software - may be modified and shared by FRC teams. The code   */
+/* must be accompanied by the FIRST BSD license file in the root directory of */
+/* the project.                                                               */
+/*----------------------------------------------------------------------------*/
+
+package frc.robot.subsystems;
+
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+
+public class Bling extends SubsystemBase {
+
+
+  private static Bling INSTANCE = new Bling();
+  /**
+   * Creates a new Bling.
+   */
+  private Bling() {
+
+
+  }
+
+  public static Bling getINSTANCE() {
```

Comment:
Typically the getINSTANCE will check for null INSTANCE (and create the instance in this method if null) before returning INSTANCE

Reply:
Oh, I see... I haven't done that for shuffleboard either, should I change it on both?